src/main/java/WayofTime/bloodmagic/api/impl/ItemSigil.java

```diff
@@ -0,0 +1,67 @@
+package WayofTime.bloodmagic.api.impl;
+
+import WayofTime.bloodmagic.api.Constants;
+import WayofTime.bloodmagic.api.iface.ISigil;
+import WayofTime.bloodmagic.api.util.helper.NBTHelper;
+import lombok.Getter;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ActionResult;
+import net.minecraft.util.EnumHand;
+import net.minecraft.world.World;
+
+/**
+ * Base class for all (static) sigils.
+ */
+public class ItemSigil extends ItemBindable implements ISigil
+{
+    @Getter
+    private int lpUsed;
+    private boolean toggleable;
+
+    public ItemSigil(int lpUsed)
+    {
+        super();
+
+        this.lpUsed = lpUsed;
+    }
+
+    @Override
+    public ActionResult<ItemStack> onItemRightClick(ItemStack stack, World world, EntityPlayer player, EnumHand hand)
+    {
+        return super.onItemRightClick(stack, world, player, hand);
+    }
+
+    public void setToggleable()
+    {
+        this.toggleable = true;
+    }
+
+    public boolean isUnusable(ItemStack stack)
+    {
+        NBTHelper.checkNBT(stack);
+
+        return stack.getTagCompound().getBoolean(Constants.NBT.UNUSABLE);
+    }
+
+    public ItemStack setUnusable(ItemStack stack, boolean unusable)
+    {
+        NBTHelper.checkNBT(stack);
+
+        stack.getTagCompound().setBoolean(Constants.NBT.UNUSABLE, unusable);
+        return stack;
+    }
+
+    public boolean getActivated(ItemStack stack)
```

Comment:
Should probably move this to `ItemSigilToggleable` and use [`IActivateable`](https://github.com/WayofTime/BloodMagic/blob/1.9/src/main/java/WayofTime/bloodmagic/api/iface/IActivatable.java).